.github/meta.yaml

```diff
@@ -56,6 +56,8 @@ outputs:
         - pyproject.toml
       commands:
         - pytest evalml/tests -n 8 --ignore evalml/tests/automl_tests/parallel_tests/ --is-using-conda
+        - python -c "from evalml.demos import load_fraud; load_fraud(nrows=10)"
```

Comment:
is this for testing purposes? If so we should remove it 😄

Reply:
No it is not for testing purposes. It is to make sure that we distribute the CSV file with evalml.